pages/common/julia.md

```diff
@@ -0,0 +1,28 @@
+# julia
+
+> A high-level, high-performance dynamic programming language for technical computing.
+> It aims to provide the ease-of-use of scripting languages with the speed of compiled languages.
```

Comment:
Just wondering if this line is really needed. I see this "pitch" being too commonly used now-a-days.